src/main/java/jenkins/plugins/git/GitSCMFileSystem.java

```diff
@@ -282,13 +282,16 @@ public SCMFileSystem build(@NonNull Item owner, @NonNull SCM scm, @CheckForNull
             if (rev != null && !(rev instanceof AbstractGitSCMSource.SCMRevisionImpl)) {
                 return null;
             }
-            TaskListener listener = new LogTaskListener(LOGGER, Level.FINE);
             GitSCM gitSCM = (GitSCM) scm;
             UserRemoteConfig config = gitSCM.getUserRemoteConfigs().get(0);
             BranchSpec branchSpec = gitSCM.getBranches().get(0);
             String remote = config.getUrl();
+            if (remote == null) {
```

Comment:
Please log a warning message that the URL in a null URL was configured int he UserRemoteConfig.  Silently returning null seems like it will need diagnosis by the administrator or the user.

Reply:
Added logging